src/vendor/Dropper.sol

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8;
```

Comment:
should definitely not be copying over the whole file. We can either install it as a git module or create an interface.

Reply:
There's a reason I did this. Any other way and we're back in dependency hell.

Reply:
If you use either Forge or Yarn to install it will not be able to compile because it maps @openzeppelin/contracts to old-oz/contracts and cannot find the file.